pkg/user/tx_client.go

```diff
@@ -205,7 +218,7 @@ func NewTxClient(
 }
 
 // SetupTxClient uses the underlying grpc connection to populate the chainID, accountNumber and sequence number of all
-// the accounts in the keyring.
+// the accounts in the keyring. By default, the client operates in trusted mode use the consensus endpoint that you trust.
```

Comment:
Again not a default because no option to override the trusted mode.

```suggestion
// the accounts in the keyring. Use a GRPC connection that you trust.
```